src/hotspot/os/windows/os_windows.cpp

```diff
@@ -382,7 +382,8 @@ void os::init_system_properties_values() {
     char path[MAX_PATH];
     char buf[2 * MAX_PATH + 2 * sizeof(EXT_DIR) + sizeof(PACKAGE_DIR) + 1];
     GetWindowsDirectory(path, MAX_PATH);
-    sprintf(buf, "%s%s;%s%s%s", Arguments::get_java_home(), EXT_DIR,
+    os::snprintf_checked(buf, sizeof(buf), "%s%s;%s%s%s",
+            Arguments::get_java_home(), EXT_DIR,
             path, PACKAGE_DIR, EXT_DIR);
```

Comment:
When the call is split across multiple lines like this, the subsequent lines should align with the first parameter i.e. align under `buf`.

Reply:
I think the other acceptable approach is to put the first argument on the 2nd line, and then indent all the argument lines by 8. That works well if the first argument would otherwise start too far to the right. For the above, I think either approach would look ok.

Reply:
OK.  I have no strong preference.  Updated to align with the 1st parameter.